readme.md

```diff
@@ -51,6 +46,29 @@ Type: `object`
 
 The `system`, `username`, `password` options must be specified together.
 
+##### verbose
+
+Type: `boolean`
+
+Return verbose results (default: `false`).
```

Comment:
Write the default value like done here: https://github.com/sindresorhus/boxen#borderstyle